cli/compose/loader/volume.go

```diff
@@ -70,6 +70,8 @@ func populateFieldFromBuffer(char rune, buffer []rune, volume *types.ServiceVolu
 		switch option {
 		case "ro":
 			volume.ReadOnly = true
+		case "rw":
```

Comment:
I think we should only support those in the expanded format. I'll open a new PR for that.